### PR TITLE
Update copyright year

### PR DESCRIPTION
**Pull request by epwinchell**

### CODE REVIEW

File: site/_includes/footer.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
   <div class="footer-row" id="footer_bottom">
     <div class="footer-legal">
       <span class="copyright">
-        &copy; 2016 ManageIQ.
+        &copy; 2017 ManageIQ.
```
Comment (chrisarcand) [inline site/_includes/footer.html]:
```
&copy; {%= Time.now.year %} ManageIQ.
```

Comment (cybette) [inline site/_includes/footer.html]: or perhaps `Date.current.year` ? we don't need the hours/minutes details and it takes time zones into consideration

Comment (skateman) [inline site/_includes/footer.html]: `{{ site.time | date: '%y' }}`

Comment (chrisarcand) [inline site/_includes/footer.html]: @cybette Time.now.year is only the year, and I wouldn't worry about timezones over the few hours on the new year :)

@skateman's solution is more Jekylly, looks good.

**Pull request merged**